src/index.js

```diff
@@ -134,7 +134,7 @@ const monitorChromecastPlayer = function (player, options) {
             }
 
             if (event.requestData.media.contentType !== undefined) {
```

Comment:
Technically we should also be comparing this by something like

```javascript
if (typeof event.requestData.media.contentType !== 'undefined') {
```

But I'm not sure it matters.